test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ]  [required]

- [ ] Please add the test.md file to the gitignore file because The content of this file is not relevant to this project